conf/distro/nilrt.conf

```diff
@@ -2,7 +2,7 @@ require nilrt.inc
 
 DISTRO_NAME = "NI Linux Real-Time"
 
-DISTRO_VERSION = "8.10"
+DISTRO_VERSION = "8.11"
 
 NILRT_FEED_NAME = "2021.5"
```

Comment:
Could you also change the `FEED_NAME` to `2021.8`, while you're at it.

Reply:
Sure, but that change shouldn't be cherry picked into nilrt/21.5/sumo right?  This change should be.  So assuming I'm right, lets make the NILRT_FEED_NAME change as another PR that is NOT cherry-picked into nilrt/21.5/sumo.  Sound good?

Reply:
The `DISTRO_VERSION` change shouldn't be cherry-picked to the release branch either. The 21.5 version is `8.10`.

Reply:
Absolutely correct.  I got confused there.  Thanks, and I've added your requested change.